### ISSUE TITLE
Animate font awesome icons

### ISSUE THREAD

**Issue by DavidCasillasRivero** [wmira/react-icons-kit#31]
Font Awesome icons support animation via `fa-spin` (rotating icon) or `fa-pulse` (rotating icon in discrete steps) classes. 
[https://fontawesome.com/v4.7.0/examples/](https://fontawesome.com/v4.7.0/examples/)

Is it possible to add this effect to React Icons Kit version of these icons?

**Comment by wmira** [comment]
Hi @DavidCasillasRivero 

Will doing something like this work for you? https://codesandbox.io/s/r4z7z64q6p

You should be able to do some if statements to change which container ( or even directly pass attribute )

**Comment by DavidCasillasRivero** [comment]
Yes I manage to add the effect adding a class with some CSS to an outer component wrapper of the icon. I don't have any animation knowledge but this copied & pasted CSS did the job for me:

```
.csl-spin {
  animation: rotate 1s linear infinite;
}

@keyframes rotate {
  to {
    transform: rotate(360deg);
  }
}
```

Just wondering if there was an easier approach  build in in your library to set the `fa-spin` class and avoid any custom CSS.

Thanks.
